### PR TITLE
Adds timeseries to page (not hardcoded)

### PR DESCRIPTION
**Pull request by briafassler9**

### CODE REVIEW

**Review by kennzoid — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 1]
```diff
@@ -41,40 +43,29 @@
...
-// TODO(briafassler): Fix the problem where chart only appears after pressing submit
 @WebServlet("/chart-data")
 public class ChartServlet extends HttpServlet {
   private static final String TABLE_NAME = "Police Killings By Year";
-  private static final String YEAR = "year";
-  private static final String AMOUNT_KILLED = "amount killed";
-  private static final String test = "{'2020': '5'}";
+  private static final String DATE = "Date";
+  private static final String TOTALS = "Totals";
+  private static final String YEAR_ORDER = "Years";
```
Comment (Jaxobc) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java]: I don't think this gets used anywhere.

**Review by Jaxobc — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 1 above)
Comment (kofimeighan) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java]: +1

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 2]
```diff
@@ -41,40 +43,29 @@
...
+    Query query = new Query(TABLE_NAME).addSort(DATE, SortDirection.ASCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
     List<DataPair> dataPairs = new ArrayList<DataPair>();
     for (Entity entity : results.asIterable()) {
-      int amountKilled = (int) entity.getProperty(AMOUNT_KILLED);
-      String year = (String) entity.getProperty(YEAR);
+      int amountKilled = ((Long) entity.getProperty(TOTALS)).intValue();
+      int year = ((Long) entity.getProperty(DATE)).intValue();
```
Comment (kofimeighan) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:61]: interesting casting method

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 3]
```diff
@@ -41,40 +43,29 @@
...
-    dataPairs.add(new DataPair(1142, "2018"));
-    dataPairs.add(new DataPair(1099, "2019"));
-    dataPairs.add(new DataPair(576, "2020"));
-
     // Send the JSON as the response
     response.setContentType("application/json");
     response.getWriter().println(new Gson().toJson(dataPairs));
   }
-}
+}
```
Comment (kofimeighan) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java]: add newline at the end of the file

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 1 above)
Comment (briafassler9) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java]: deleted

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 2 above)
Comment (briafassler9) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:61]: gotta give credit to kenneth, he helped me out

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 3 above)
Comment (briafassler9) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java]: added

**Pull request merged**